Backends/CLX-fb/mirror.lisp

```diff
@@ -109,7 +103,8 @@
   (clim-clx::port-set-mirror-transformation port (slot-value mirror 'xmirror) mirror-transformation))
 
 (defun %mirror-force-output (mirror)
-  (check-type mirror clx-fb-mirror)
+  (unless (typep mirror 'clx-fb-mirror)
+    (return-from %mirror-force-output))
```

Comment:
Can we instead ensure that this is only called with proper arguments?

Reply:
That is, can we keep the previous version?

Reply:
OK (fixed).